javascript/javascript1/week2/Candy.js

```diff
@@ -0,0 +1,73 @@
+// Candy helper optional 
+
+
+const boughtCandyPrice = []; 
+function addCandy( candyType , weight){
+   
+   let firstType ="Sweet";
+   let  secondType ="Chocolate";
+   let thirdType ="Toffee";
+   let fourthType = "Chewing-gum";
+    let price = 0; 
+    //const amountToSpend = Math.random() * 100; 
+
+    if ( firstType === "Sweet"){
```

Comment:
Nit: probably you want to use candyType variable here and in other if-else-ifs? 
because in current code firstType  variable is equal to "Sweet", so this if always works, putting "Sweets" into array of candies. 
And arguments of the function does not matter

Reply:
Thanks for  the comments Anton  . i will check them :).